packages/client/src/pages/settings/index.vue

```diff
@@ -34,241 +34,229 @@ import * as symbols from '@/symbols';
+		case 'email': return defineAsyncComponent(() => import('./email.vue'));
+		case 'theme': return defineAsyncComponent(() => import('./theme.vue'));
+		case 'theme/install': return defineAsyncComponent(() => import('./theme.install.vue'));
+		case 'theme/manage': return defineAsyncComponent(() => import('./theme.manage.vue'));
+		case 'menu': return defineAsyncComponent(() => import('./menu.vue'));
+		case 'sounds': return defineAsyncComponent(() => import('./sounds.vue'));
+		case 'custom-css': return defineAsyncComponent(() => import('./custom-css.vue'));
+		case 'deck': return defineAsyncComponent(() => import('./deck.vue'));
+		case 'plugin': return defineAsyncComponent(() => import('./plugin.vue'));
+		case 'plugin/install': return defineAsyncComponent(() => import('./plugin.install.vue'));
+		case 'import-export': return defineAsyncComponent(() => import('./import-export.vue'));
+		case 'account-info': return defineAsyncComponent(() => import('./account-info.vue'));
+		case 'delete-account': return defineAsyncComponent(() => import('./delete-account.vue'));
+	}
+	return null;
+});
 
-		watch(component, () => {
-			pageProps.value = {};
+watch(component, () => {
+	pageProps.value = {};
 
-			nextTick(() => {
-				scroll(el.value, { top: 0 });
-			});
-		}, { immediate: true });
+	nextTick(() => {
+		scroll(el.value, { top: 0 });
+	});
+}, { immediate: true });
 
-		watch(() => props.initialPage, () => {
-			if (props.initialPage == null && !narrow.value) {
-				page.value = 'profile';
-			} else {
-				page.value = props.initialPage;
-				if (props.initialPage == null) {
-					INFO.value = indexInfo;
-				}
-			}
-		});
+watch(() => props.initialPage, () => {
+	if (props.initialPage == null && !narrow.value) {
+		page.value = 'profile';
+	} else {
+		page.value = props.initialPage;
+		if (props.initialPage == null) {
+			INFO.value = indexInfo;
+		}
+	}
+});
 
-		onMounted(() => {
-			narrow.value = el.value.offsetWidth < 800;
-			if (!narrow.value) {
-				page.value = 'profile';
-			}
-		});
+onMounted(() => {
+	narrow.value = el.value.offsetWidth < 800;
+	if (!narrow.value) {
+		page.value = 'profile';
+	}
+});
 
-		const emailNotConfigured = computed(() => instance.enableEmail && ($i.email == null || !$i.emailVerified));
+const emailNotConfigured = computed(() => instance.enableEmail && ($i.email == null || !$i.emailVerified));
 
-		const pageChanged = (page) => {
-			if (page == null) return;
-			childInfo.value = page[symbols.PAGE_INFO];
-		};
+const pageChanged = (page) => {
+	if (page == null) return;
+	childInfo.value = page[symbols.PAGE_INFO];
+};
 
-		return {
-			[symbols.PAGE_INFO]: INFO,
-			page,
-			menuDef,
-			narrow,
-			view,
-			el,
-			pageProps,
-			component,
-			emailNotConfigured,
-			pageChanged,
-			childInfo,
-		};
-	},
+defineExpose({
+	[symbols.PAGE_INFO]: INFO,
+	page,
+	menuDef,
+	narrow,
+	view,
+	el,
+	pageProps,
+	component,
+	emailNotConfigured,
+	pageChanged,
+	childInfo,
```

Comment:
When using setup suger, defineExpose is used to define the values you want to expose "outside the component".
So I think the only thing you actually need to expose here is `[symbols.PAGE_INFO]`.